misc/vim/plugin/nit.vim

```diff
@@ -243,13 +243,61 @@ fun NitOmnifunc(findstart, base)
 		if !handled
 			" It may be a keyword
 			if strlen(a:base) > 0
-				for keyword in ['module', 'import', 'class', 'abstract', 'interface',
-					\'universal', 'enum', 'end', 'fun', 'type', 'init', 'redef', 'is',
-					\'do', 'var', 'extern', 'public', 'protected', 'private', 'intrude',
-					\'if', 'then', 'else', 'while', 'loop', 'for', 'in', 'and', 'or',
-					\'not', 'implies', 'return', 'continue', 'break', 'abort', 'assert',
-					\'new', 'isa', 'once', 'super', 'self', 'true', 'false', 'null',
-					\'as', 'nullable', 'isset', 'label']
+				for keyword in [
+					\'abort',
+					\'abstract',
+					\'and',
+					\'as',
+					\'assert',
+					\'break',
+					\'catch',
+					\'class',
+					\'continue',
+					\'__debug__',
+					\'do',
+					\'else',
+					\'end',
+					\'enum',
+					\'extern',
+					\'false',
+					\'for',
+					\'fun',
+					\'if',
+					\'implies',
+					\'import',
+					\'in',
+					\'init',
+					\'interface',
+					\'intrude',
+					\'is',
+					\'isa',
+					\'isset',
+					\'label',
+					\'loop',
+					\'module',
+					\'new',
+					\'not',
+					\'null',
+					\'nullable',
+					\'once',
+					\'or',
+					\'package',
+					\'private',
+					\'protected',
+					\'public',
+					\'redef',
+					\'return',
+					\'self',
+					\'subset',
+					\'super',
+					\'then',
+					\'true',
+					\'type',
+					\'universal',
+					\'var',
+					\'while',
+					\'with',
+					\'yield']
```

Comment:
I first tried to minimize the changes as you said, but since each definition had its own order and not the same missing/obsolete keywords, it was very tedious. By fixing the order, I only have to do a search-replace on the grammar to translate the list.